### PR TITLE
add fip proposing p256 address types

### PR DESCRIPTION
**Pull request by whyrusleeping**
I've wanted this for a while, seems like better late than never.

### CODE REVIEW

File: FIPS/fip-p256-keys.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the FIP.-->
+Add a new address type backed by NIST p256 keys, allowing the majority of existing secure enclaves (including iPhones and most android phones). This would allow very easy interactions with the Filecoin network by (for example) allowing users to send FIL securely using apple faceID or other secure enclave methods
+
+## Abstract
+Filecoin ramps should be as simple and available as possible, adding p256 opens up a whole new set of solutions for developers to integrate with.
+
+## Change Motivation
```
Comment (Stebalien) [inline FIPS/fip-p256-keys.md:21]: From what I can tell, the iPhone also supports Curve25519 (https://developer.apple.com/documentation/cryptokit/curve25519) and the docs specify that the enclave supports "256bit elliptic curve algorithms". Are you sure it only supports P256?

Unfortunately, it looks like Android may only support RSA and P256.

Maybe we should just add support for both at the same time?

Comment (Stebalien) [reply FIPS/fip-p256-keys.md:21]: The cost of adding two algorithms versus one is pretty minimal.

Comment (whyrusleeping) [reply FIPS/fip-p256-keys.md:21]: Yeah, the more the merried, Curve25519 is getting a lot of traction too, and youre right, adding more is easy.

**Comment by kaitlin-beegle** [comment]
Hey @whyrusleeping, would you mind adding this proposal/summary to the [FIPs Discussion Forum](https://github.com/filecoin-project/FIPs/discussions) too, so I can steward it for community feedback?

Thank you!

**Comment by kaitlin-beegle** [comment]
Reached out again to FIP author to determine whether to merge or deprecate this draft.

**Comment by kaitlin-beegle** [comment]
Closing PR, per [this thread. ](https://filecoinproject.slack.com/archives/CSC2632KB/p1657688905609929)

**Pull request closed without merging**